### PR TITLE
Add detail to error logs

### PR DESCRIPTION
**Pull request by rebeccacremona**
Scoop is designed with [`loglevel`](https://www.npmjs.com/package/loglevel)'s debugging strategy in mind:
> disable all but error logging in production, and then run log.setLevel("trace") in your console to turn it all back on for a furious debugging session

This works great when running manually as a standalone utility, but is proving a little challenging when using Scoop as part of an orchestrated system, as we do with Perma.cc, which calls to a REST API that invokes Scoop: we occasionally see errors that we cannot reproduce locally on demand, and, because we lack the trace, we don't have a good sense of what's going wrong.

This PR attempts to sneak a reasonable amount of debugging information into the `ERROR` level logs, without cluttering them or compromising the logging philosophy.

It hopefully does not break anything.

It may not yet hit the sweet spot: it could be, the resulting log lines will prove messy. But, until I see some, I am not sure what else to strip away 🙂

### CODE REVIEW

File: utils/formatErrorMessage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    message = err.message
+  } else if (err instanceof String) {
+    message = err
+  } else {
+    message = String(err)
+  }
+
+  // strip ansi
+  // https://stackoverflow.com/a/29497680
+  const ansiRegex = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g
```
Comment (matteocargnelutti) [inline utils/formatErrorMessage.js:22]: @rebeccacremona Please don't hesitate to add `//eslint-disable-line` so Standard JS stops barking at this 😄 .

**Comment by matteocargnelutti** [comment]
(I know this is a draft but: Big fan 🤩  )

File: utils/formatErrorMessage.js (hunk 1 above)
Comment (rebeccacremona) [reply utils/formatErrorMessage.js]: Ah! Thanks for the tip!

File: utils/formatErrorMessage.test.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+test('formatErrorMessage should truncate length', async (_t) => {
+  try {
+    throw new Error('a'.repeat(200))
+  } catch (err) {
+    assert(formatErrorMessage(err) === 'a'.repeat(100) + '...')
+  }
+})
+
+test('formatErrorMessage should strip block formatting', async (_t) => {
+  const actualErrorMessage = `
```
Comment (matteocargnelutti) [inline utils/formatErrorMessage.test.js:23]: Up to you: You could move this into a text file under `/assets/fixtures` to (marginally) improve readability / maintainability.

**Review by matteocargnelutti — APPROVED** [review]
🤘

File: utils/formatErrorMessage.test.js (hunk 2 above)
Comment (rebeccacremona) [reply utils/formatErrorMessage.test.js]: 👍 done!

**Pull request merged**